### ISSUE TITLE
Undeclared constant in UpdateClient class

### ISSUE THREAD

**Issue by xxsimoxx** [codepotent/Update-Manager#24]
# Description 
A warning appears when clicking on "View details" on any plugin using Update Manager (but not on Update Manager itself).
```
PHP Warning:  Use of undefined constant PLUGIN_PREFIX - assumed 'PLUGIN_PREFIX' (this will throw an Error in a future version of PHP) in .../stats-for-update-manager/classes/UpdateClient.class.php on line 565PHP message: PHP Warning:  Use of undefined constant PLUGIN_PREFIX - assumed 'PLUGIN_PREFIX' (this will throw an Error in a future version of PHP) in .../stats-for-update-manager/classes/UpdateClient.class.php on line 566
```

# Steps to Reproduce
1. Go to plugin screen
2. Click on View details on a plugin using the new class

# Expected Behavior
No warnings

# Screenshots
If applicable, add screenshots to help explain the problem.
![image](https://user-images.githubusercontent.com/29772709/71973201-9b05c900-320e-11ea-9261-b9db74d008d6.png)

# Additional Context
The constants used [here](https://github.com/codepotent/Update-Manager/blob/1b0552d146eea55c5b802aebe4ce2d65ae0e0cdb/classes/UpdateClient.class.php#L557-L558) are declared in `includes/constants.php`.

**Comment by xxsimoxx** [comment]
Proposed a fix in PR#25.

**Comment by johnalarcon** [comment]
Looks like you were drafting this while I was drafting the new release. Really appreciate your quick response to this. Got it handled and the fix has been pushed in release 1.0.1. :)
